README.md

```diff
@@ -196,6 +196,20 @@ In some cases it's needed to tell visual test to wait before doing a snapshot. I
 
 The list of all imported icons is in `/tools/icongen.config`. After modifying the list you should run `npm run icongen` or it will be run on prepush.
 
+### Making a Release
+
+All releasable code is aggregated in the `develop` branch. To make a release simply create a GitHub release (usually copying over the current entries from the `CHANGELOG.md` file).
```

Comment:
> From the name itself develop is usually used only for developing or testing purposes and it might be a bit confusing.

It still is. This is the reason for using `develop`. `develop` produces a preview version - as it does now.

> Why not use master branch instead to aggregate and deploy changes?

The name is outdated and should no longer be used.

Reply:
`develop` produces a preview version true. But it will also be used to make the release, right?
We could go for `main` instead.

Reply:
Sure - we can do that.

Reply:
I've renamed `develop` to `main`. Once this PR is in I'd recommend to remove `master` / make a cleanup. Many of the PRs should then also be (auto) closed as the vulnerability should be gone.